GVFS/GVFS.FunctionalTests/Tools/ControlGitRepo.cs

```diff
@@ -55,7 +55,7 @@ public void Initialize()
             GitProcess.Invoke(this.RootPath, "config merge.renames false");
             GitProcess.Invoke(this.RootPath, "config advice.statusUoption false");
             GitProcess.Invoke(this.RootPath, "config core.abbrev 40");
-            GitProcess.Invoke(this.RootPath, "config core.useBuiltinFSMonitor false");
+            GitProcess.Invoke(this.RootPath, "config core.fsmonitor false");

```

Comment:
@jeffhostetler: Perhaps the old line should stay here, and we just _add_ the `core.fsmonitor` line, too. Keeping the old config won't hurt, and might help for users still on an older version of Git.